docs/docs/production-app.md

```diff
@@ -113,7 +113,7 @@ PageRenderer's constructor [loads the page resources](/docs/production-app/#load
 
 Before hydration occurs, we kick off the loading of resources in the background. As mentioned above, the current page's resources will have already been requested by `link` tags in the HTML. So, technically, there's nothing more required for this page load. But we can start loading resources required to navigate to other pages.
 
-This occurs in [loader.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js). The main function here is [getResourcesForPathname()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L268). Given a path, it will find its page, and import its component module json query results. But to do this, it needs access to that information. This is provided by [async-requires.js](http://localhost:8000/docs/write-pages/#async-requiresjs) which contains the list of all pages in the site, and all their dataPathss. [fetchPageResourcesMap()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L33) takes care of requesting that file, which occurs the [first time](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L292) `getResourcesForPathname()` is called. 
+This occurs in [loader.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js). The main function here is [getResourcesForPathname()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L268). Given a path, it will find its page, and import its component module json query results. But to do this, it needs access to that information. This is provided by [async-requires.js](http://localhost:8000/docs/write-pages/#async-requiresjs) which contains the list of all pages in the site, and all their dataPathss. [fetchPageResourcesMap()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L33) takes care of requesting that file, which occurs the [first time](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L292) `getResourcesForPathname()` is called.
```

Comment:
Hey @jonsuh, thanks for helping out! 👋 

This paragraph contains one more `localhost:8000`, could you please fix that one, too? 🙏

Reply:
@fk Sure, thanks for catching that! Apparently overlooked it. Fixed in https://github.com/gatsbyjs/gatsby/pull/8276/commits/10c979e4360ebbf08ae10fffd3436d062da9f75e

Reply:
Thank you Jonathan! 🙏